index.html

```diff
@@ -10,7 +10,10 @@
   <body>
     <div id="app">
       <div data-view="catalog"></div>
-      <div data-view="details"></div>
+      <div data-view="details">
+        <div id="cart"></div>
```

Comment:
This cart element should be visible in the catalog too. I would just move it outside for the `data-view` containers.